Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -382,15 +382,24 @@ void VertexShaderManager::SetConstants()
     // NOTE: If we ever emulate antialiasing, the sample locations set by
     // BP registers 0x01-0x04 need to be considered here.
     const float pixel_center_correction = 7.0f / 12.0f - 0.5f;
-    const float pixel_size_x = 2.f / g_renderer->EFBToScaledXf(2.f * xfmem.viewport.wd);
-    const float pixel_size_y = 2.f / g_renderer->EFBToScaledXf(2.f * xfmem.viewport.ht);
+    const float viewport_width = g_ActiveConfig.bVertexRounding ?
```

Comment:
Actually, re-looked at this after the change and realized I probably didn't need that here.  The "else" case for if bVertexRounding is false, is calling EFBToScaledXf which at 1x should yield the same result.  No biggie either way, just thought I'd mention it.